device/mellanox/x86_64-mlnx_msn4700-r0/Mellanox-SN4700-O28/sai.profile

```diff
@@ -0,0 +1,3 @@
+SAI_INIT_CONFIG_FILE=/usr/share/sonic/hwsku/sai_4700.xml
+SAI_DUMP_STORE_PATH=/var/log/mellanox/sdk-dumps
+SAI_DUMP_STORE_AMOUNT=10
```

Comment:
Please sync with the recent changes of another SN4700 of store and forward and update it. check as a reference the changes done in https://github.com/sonic-net/sonic-buildimage/pull/16781

Reply:
Fixed in https://github.com/sonic-net/sonic-buildimage/pull/16784/commits/64761e9ad3f008c9ed3fd1e657b6121622fcb174